.changeset/solid-socks-ask.md

```diff
@@ -0,0 +1,11 @@
+---
+'@sku-lib/codemod': major
+'@sku-lib/create': major
+---
+
+Update commander.js to the latest version.
```

Comment:
Should the release note be more focused on the impact rather than the change?

```suggestion
Throw error when given unsupported arguments.
This is now the default behaviour of the updated commander.js version.
```